src/auth/router.mjs

```diff
@@ -33,6 +33,61 @@ authRouter.post("/login", async (req, res) => {
   }
 });
 // Ruta protegida de ejemplo
-authRouter.get("/protected", verifyToken, checkRole(["ADMIN"]), async (req, res) => {
-  res.json({ message: "Access granted", user: req.user });
+authRouter.get(
+  "/protected",
+  verifyToken,
+  checkRole(["ADMIN"]),
+  async (req, res) => {
+    res.json({ message: "Access granted", user: req.user });
+  }
+);
+authRouter.post("/sign-up", async (req, res, next) => {
+  const { name, lastName, email, password, phone } = req.body;
+  if (!name || !/^[a-zA-Z]+(\s[a-zA-Z]+)*$/.test(name)) {
+    return res.status(400).json({
+      error: "El nombre es obligatorio y no puede incluir números ni símbolos",
+    });
+  }
+  if (!lastName || !/^[a-zA-Z]+(\s[a-zA-Z]+)*$/.test(lastName)) {
```

Comment:
no es bloqueante, pero como to-do: podriamos guardar la regex en una variable asi no la escribimos dos veces